com.unity.cinemachine/Samples~/Timeline Sample/ActiveControlsWhenTimelineStops.cs

```diff
@@ -0,0 +1,23 @@
+using System;
```

Comment:
This is very confusing.  It's not clear what is being demoed here.  I think this scene can be replaced by a sample that demonstrates a cutscene.

Reply:
This scene is still horrible.  The help helps but it doesn't help enough to really help.  We need to rethink this scene from the ground up.